middle_end/flambda2/types/meet_and_join.ml

```diff
@@ -386,11 +390,13 @@ and meet_head_of_kind_value env (head1 : TG.head_of_kind_value)
           alloc_mode = alloc_mode2
         } ) ->
     let<* alloc_mode = meet_alloc_mode alloc_mode1 alloc_mode2 in
-    let<* element_kind = meet_array_element_kinds element_kind1 element_kind2 in
+    let element_kind = meet_array_element_kinds element_kind1 element_kind2 in
     let<* contents, env_extension =
       meet_array_contents env array_contents1 array_contents2
     in
     let<* length, env_extension' = meet env length1 length2 in
+    (* Note: If the element kind is Bottom, we could meet the length type with
```

Comment:
This should probably be a `CR-someday`.  Missing parenthesis at the end of the comment too.

Reply:
Done